### DISCUSSION TITLE
can't resize image on preview mode/reading mode

### DISCUSSION THREAD

**Discussion by iamchiccc** [gavvvr/obsidian-imgur-plugin#62] [Getting help]
I can resize image that I upload in editing mode, but when I toggle on reading mode the image appears in the original size. How can I fix this?

**Comment by gavvvr** [comment]
Hi @gayoungmoney 
Not sure what you meanе by 
> "I can resize image that I upload in editing mode"

You can only see images in "Live Preview" type of editing mode, you probably meant that. At the same time, I have no idea how to "resize" in Live Preview mode, so I did not get you.

Meanwhile, I would like to draw your attention, that the Imgur-uploaded image is not somehow special. It's just a remote image embedded to your note with Markdown, like any other remote image. It can be resized with any other plugin which has this feature. As I've mentioned in [another discussion](https://github.com/gavvvr/obsidian-imgur-plugin/discussions/58#discussioncomment-7564827) you can for example use [mousewheel-image-zoom](https://github.com/nicojeske/mousewheel-image-zoom) to resize image.
